cob_mimic/src/mimic_node.cpp

```diff
@@ -75,12 +82,11 @@ class Mimic
             "-q",
             "--no-osd",
             "-L",
-            "--one-instance",
+            "--no-one-instance",
             "--playlist-enqueue",
             "--no-video-title-show",
             "--no-skip-frames",
             "--no-audio",
-            "--vout=glx,none"
```

Comment:
what do these arg changes do?

Reply:
These arg changes where necessary to get it running under melodic. Because the node is running stable on melodic with this changes, we kept them here as well. 
- one-instance: Allow only one running instance
- vout: default video output mode => glx,none = opengl via glx
  if it is not defined, vlc will autopick the best option

Reply:
thanks

 - so now we allow more than one instances - the flag is now set to `--no-one-intance`???
 - is the "best option" in our setup something different then `glx,none`? does the auto-select feature improve performance by chance?

Reply:
> * so now we allow more than one instances - the flag is now set to `--no-one-intance`??? in what case do we need this?

I think, that `one-instance` is the right option we need, because we do not like to open another window for a new video. Switching to `no-one-instance` was only done, because `one-instance` was not working on melodic. @HannesBachter  can you test it with the original option?



> * is the "best option" in our setup something different then `glx,none`? does the auto-select feature improve performance by chance?

`glx,none` was the only option that worked on older robots. We should either keep the option for now or test if it works on all our robots without the explicit vout option

Reply:
tested with the old parameters for vlc and did not crash then, too